scripts/templates/otel_summary.j2

```diff
@@ -0,0 +1,30 @@
+{%import 'macros.j2' as macros%}
+The following table summarizes the alignment status by namespaces between ECS in version `{{ ecs_generated_version }}`
+and OpenTelemetry semantic conventions in version `{{ semconv_version }}`.
+
+[cols="8*"]
+|====
+h| 
+7+^h| Number of Fields / Attributes
+
+h| Namespace
+h| ECS
+h| OTel 
```

Comment:
```suggestion
h| https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html[ECS]
h| https://opentelemetry.io/docs/specs/semconv/attributes-registry[OTel]
```

Reply:
Alternatively, maybe add an ECS/OTel icon to the `summary['title']` column and link to the corresponding namespace.